editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -66,16 +68,15 @@ void EditorAssetLibraryItem::configure(const String &p_title, int p_asset_id, co
 
 // TODO: Refactor this method to use the TextServer.
 void EditorAssetLibraryItem::clamp_width(int p_max_width) {
-	int text_pixel_width = title->get_button_font().ptr()->get_string_size(title->get_text()).x * EDSCALE;
-
-	String full_text = title->get_text();
-	title->set_tooltip_text(full_text);
+	int text_pixel_width = title->get_button_font().ptr()->get_string_size(title_text).x * EDSCALE;
```

Comment:
```suggestion
	int text_pixel_width = title->get_button_font()->get_string_size(title_text).x * EDSCALE;
```
This can be removed.